### DISCUSSION TITLE
Is there a way to use cucumber tags in allure reporting and categorise based on these tags

### DISCUSSION THREAD

**Discussion by varune** [allure-framework/allure2#2016] [General Discussion]
We have tags mentioned for scenarios in cucumber feature files.
Now we are looking to show results based on these tags in execution results report.
Currently, we have integration with allure report and we are looking to have results categorised based on labels.
Any suggestions?

**Comment by bhala-ki** [comment]
Yes, this is really required as to segregate the tests based on the tags (cucumber) & report the metrics of how many passed / failed. We have the allure report with the tags embedded on the scenario/test, it would be good to click on the tag & aggregate/group all the tests tagged as that tag to view the report. Thanks!

![image](https://github.com/allure-framework/allure2/assets/139777339/3be432f8-a345-478c-a5dd-40835e1f24c1)

Basically anchor the tags to its tests.
